src/monorepo-workflow-operations.test.ts

```diff
@@ -40,42 +67,6 @@ async function fileExists(entryPath: string): Promise<boolean> {
   }
 }
 
-/**
- * Mocks the dependencies for `followMonorepoWorkflow`.
- *
- * @returns The corresponding mock functions for each of the dependencies.
- */
-function getDependencySpies() {
```

Comment:
Writing out the return type for this function was going to be a huge chore, so I refactored the setup of these mocks.